matrix/common/dynamo_handler.py

```diff
@@ -68,27 +71,34 @@ def _get_dynamo_table_resource_from_enum(self, dynamo_table: DynamoTable):
         elif dynamo_table == DynamoTable.OUTPUT_TABLE:
             return self._output_table
 
-    def create_state_table_entry(self, request_id: str, num_bundles: int, format: str=MatrixFormat.ZARR.value):
+    def create_state_table_entry(self,
+                                 request_id: str,
+                                 num_mappers: int,
+                                 num_workers: int,
+                                 format: str=MatrixFormat.ZARR.value):
         """
         Put a new item in the DynamoDB table responsible for tracking task execution states and
-        counts for a specified job.
+        counts for a specified request.
 
         :param request_id: UUID identifying a filter merge job request.
-        :param num_bundles: Number of bundles to be processed.
+        :param num_mappers: Number of mapper lambdas expected to be invoked.
+        :param num_workers: Number of worker lambdas expected to be invoked.
```

Comment:
I'm not sure the driver is going to know the number of worker lambdas to be invoked once we have other bundle types.

Reply:
Ah, because we'll chunk out each mapper differently based on its load yea? I'll change it back then, was just trying to clean up